cypress/tests/resource-provider-card.spec.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  PVC_NAME,
+  DATA_SOURCE_NAME_NSFS,
+  DATA_SOURCE_NAME_AWS,
+} from '../constants/tests';
+import { dataSourceAWS, dataSourceNSFS } from '../mocks/data-source';
+import { MCGMSCommon } from '../views/mcg-ms-common';
+
+describe('admin dashboard - resource provider card', () => {
+  before(() => {
+    cy.login();
+  });
+
+  after(() => {
+    cy.logout();
```

Comment:
please add a clean-up of the data source also here. can use already existing methods from the data source tests